apps/vaporgui/TabManager.cpp

```diff
@@ -94,6 +94,7 @@ void TabManager::SetActiveRenderer(string activeViz, string renderClass, string
 
 void TabManager::ShowRenderWidget(string subTabName)
 {
+    cout << "MoveToFront " << subTabName << endl;
```

Comment:
Shall we leave this print output to Master branch?

Reply:
My bad!